release-notes/master.adoc

```diff
@@ -21,6 +21,8 @@ include::topics/controller-420.adoc[leveloffset=+2]
 [[anchor-february-2022-release]]
 == February 2022 release
 
+include::topics/aap-213.adoc[leveloffset=+2]
```

Comment:
Am I mistaken in thinking that `topics/aap-212.adoc` is going to need to be in here as well?  Or are we planning on waiting on the merge of that PR and then resolving the conflict?

Reply:
Hi @jbradberry : You're right. The AAP 2.1.2 release notes are being tracked in PR #[552](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/552), and will be available once the review is complete and PR is merged.